scripts/winesapos_convert.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# Script by @GuestSneezeOSDev
+echo "Convering system to winesapOS in 3"
+sleep 1
+echo "Convering system to winesapOS in 2"
+sleep 1
+echo "Convering system to winesapOS in 1"
+sleep 1
+echo "System is converting ..."
+pacman -Sy --noconfirm
+useradd -m winesap
+echo "winesap:winesap" | chpasswd
+pacman -S git wget flatpak base-devel --noconfirm
+wget https://winesapos.lukeshort.cloud/repo/iso/winesapos-4.1.0/_test/winesapos-4.1.0-beta.1-minimal-rootfs.tar.zst
```

Comment:
it should be extracted instead of being installed

Reply:
No, that would cause lots of issues if you extracted it. Installing the packages like you have down below is good enough. The only way to get a perfect 1:1 of winesapOS is to actually install it. The conversion script can be thought of as being "close enough." ;-)

Reply:
Thanks